### PR TITLE
WSGI additional args

### PR DESCRIPTION
**Pull request by atarkowska**
Updating doc regarding https://github.com/openmicroscopy/openmicroscopy/pull/4352 to show how to use additional wsgi args

### CODE REVIEW

**Comment by will-moore** [comment]
http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/install-web/install-nginx.html Looks good, but the 'property' link to omero.web.wsgi_args is not working (and I don't see it on http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/config.html).

**Comment by hflynn** [comment]
@will-moore staging build is running again now with the autogen branch included this time. You should be able to see updated config staging page in about 25 mins.

**Comment by hflynn** [comment]
http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/config.html#std:property-omero.web.wsgi_args is indeed working now.
An autogen PR will be needed once the code PR is merged to propagate this into the docs properly.

**Comment by atarkowska** [comment]
As https://github.com/openmicroscopy/ome-documentation/pull/1392 is no longer conflicting could we merge this please?

**Comment by aleksandra-tarkowska** [comment]
http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/config.html#omero-web-wsgi-args and http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/install-web/install-nginx.html are not showing appropriate links

**Comment by atarkowska** [comment]
http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/config.html#omero-web-wsgi-args and http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/install-web/install-nginx.html are now showing appropriate links

File: omero/developers/Web/Deployment.txt [hunk 1]
```diff
@@ -82,6 +82,7 @@ listening on 127.0.0.1:4080 that will autoreload on source change:
 
     $ nginx -c $PWD/nginx-development.conf
     $ bin/omero config set omero.web.application_server.max_requests 1
-    $ bin/omero web start  --wsgi-args ' --reload'
+    $ bin/omero config set omero.web.wsgi_args -- "--reload".
```
Comment (will-moore) [inline omero/developers/Web/Deployment.txt]: Shouldn't finish the line with a `.`

Comment (atarkowska) [inline omero/developers/Web/Deployment.txt]: done

File: omero/sysadmins/unix/install-web/install-nginx.txt [hunk 2]
```diff
@@ -85,9 +85,15 @@ Additional settings can be configured from command line arguments:
...
     The maximum number of simultaneous clients.
 
-.. option:: --wsgi-args WSGI_ARGS
+.. note::
+    Additional wsgi arguments can be set using property
+    :property:`omero.web.wsgi_args`
+
+    ::
+
+        $ bin/omero config set omero.web.wsgi_args -- "--forwarded-allow-ips=127.0.0.1".
```
Comment (will-moore) [inline omero/sysadmins/unix/install-web/install-nginx.txt]: Full-stop here too

**Comment by will-moore** [comment]
Other than tiny . fixes, looks good.

File: omero/sysadmins/unix/install-web/install-nginx.txt (hunk 2 above)
Comment (atarkowska) [inline omero/sysadmins/unix/install-web/install-nginx.txt]: Done

**Comment by atarkowska** [comment]
@will-moore is there anything else you spot here?

**Comment by will-moore** [comment]
Looks good now, thanks

**Pull request merged**